src/condor_tools/htcondor_cli/dagman.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, logger, dag_filename, **options):
 
         # Submit the DAG to the local schedd
         submit_description = htcondor.Submit.from_dag(dag_filename)
+	#Set s_method to HTC_DAG_SUBMIT
+        submit_description.setSubmitMethod(5)
```

Comment:
can we get the constant from the bindings here, instead of hard-coding a number?